### DISCUSSION TITLE
custom operators in Python

### DISCUSSION THREAD

**Discussion by gro1729** [MilesCranmer/PySR#716] [Q&A]
I have a huge library of operators, all of them implemented in Python. Is it possible to use these functions or do I have to reimplement them in Julia?

**Comment by gro1729** [comment]
What I have forgotten to mention: My Python- Operators have a signature of at least three parameters, like ,,age'', ,,gender'', ,,term of premium payments'', ....  . Is it possible to use PySR, SymbolicRegression.jl respectively, in order to be able to find formulas for premium calculations.

**Comment by MilesCranmer** [comment]
You can try to use PythonCall.jl to call back into Python. But it will be much slower due to Python’s GIL which blocks parallelism. It is better to reimplement in Julia.

Operators are only arity 1 and 2.

**Reply by gro1729** [reply]
Thank you very much for your reply.
